aait/backend/group-2A/Application/Features/Follow/Handlers/Command/DeleteFollowCommandHandler.cs

```diff
@@ -21,12 +24,24 @@ public DeleteFollowCommandHandler(IUnitOfWork unitOfWork, IMapper mapper)
             _mapper = mapper;
```

Comment:
remove mapper

Reply:
but i use mapper on line 35 to map followDto to follow entity